README.md

```diff
@@ -306,6 +306,8 @@ torchtune provides well-tested components with a high-bar on correctness. The li
 
 We really value our community and the contributions made by our wonderful users. We'll use this section to call out some of these contributions! If you'd like to help out as well, please see the [CONTRIBUTING](CONTRIBUTING.md) guide.
 
+- [@SalmanMohammadi](https://github.com/salmanmohammadi) for adding a comprehensive end-to-end recipe for [PPO](recipes/lora_dpo_single_device.py) training to torchtune
```

Comment:
We need a better way to handle this. These are definitely some of the best contributions we have, but we should highlight everyone who contributes to the repo. What about axolotl's acknowledgements: https://github.com/axolotl-ai-cloud/axolotl#contributing-?

Reply:
I don't intend for this to be an exhaustive list, but rather a more high-signal place to list the most valuable contributions we've got. The axolotl approach works well for them, but in our case it'll be 50% various PyTorch devs which imo defeats the purpose. I agree it's not an approach we should continue indefinitely, but this gives direct access to a GitHub user profile which I think is more valuable (on the Axolotl page it's basically just a JPEG with a hyperlink to the contributors page). We can consider keeping only the ~5 latest community contributions here so it doesn't become overwhelming though

Reply:
Awesome, thanks for volunteering to keep this list updated at all times!

Reply:
That came off snarkier than I intended

Reply:
Or did it...

Reply:
It may also introduces some weirdness/politics in who decides what is important enough.

Reply:
All contributors with the name Joe automatically go to the top of this list

Reply:
Anyone named Evan is permabanned

Reply:
> It may also introduces some weirdness/politics in who decides what is important enough.

Yeah totally agreed here. IMO PPO is some of my worst work on the repo. Why can't we call out when I [broke the checkpointer for gemma](https://github.com/pytorch/torchtune/pull/1168) and then [fixed it for sweet sweet github stats](https://github.com/pytorch/torchtune/pull/1064), or [pYdOcLiNt](https://github.com/pytorch/torchtune/pull/1240)??

Maybe we can have a monthly vote on Discord for PRs to be added. It's the community contributions, why not put it in the community's hands?

Reply:
Another idea would be to feature all contributors in the CONTRIBUTING.md file and just do a better job highlighting our added features on a monthly basis, with an emphasis on community contributions.

This would be like an "Updates" section that holds a memory of 1-3 months or so.

Reply:
> This would be like an "Updates" section on the README that holds a memory of 1-3 months or so.

https://github.com/pytorch/torchtune/pulls?q=merged%3A%3E2024-05-01+is%3Apr+sort%3Areactions+